src/EditorFeatures/CSharpTest/Completion/CompletionProviders/ObjectInitializerCompletionProviderTests.cs

```diff
@@ -612,6 +612,49 @@ End Property
                 hideAdvancedMembers: false);
         }
 
+        [WorkItem(4754, "https://github.com/dotnet/roslyn/issues/4754")]
```

Comment:
Can you add tests where the member is protected, and you either are/aren't in a subclass that has access to them?